### PR TITLE
CLOUD-2178: Flexibility, Describe how to copy data from one instance …

### PR DESCRIPTION
**Pull request by jagirard**
…to the other

**Description**

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: cloud_edition/flexibility_mode/docs/environments_access.rst [hunk 1]
```diff
@@ -102,3 +102,44 @@ If **akeneo**, as an SSH user or as a PIM process, creates files in the SFTP sub
...
+            > **`project-staging.akeneo.cloud.com`**
+
+    👨‍💼 💻   ──────→ 🔑  ──────→ 🖥  ──────→ 📂 ──────→ 🖥
+        
+      
+
+
+**Prerequisites:**
+
+- Get ssh key access to intance for akeneo user.
```
Comment (LudovicTOURMAN) [inline cloud_edition/flexibility_mode/docs/environments_access.rst:128]:
```suggestion
- Get ssh key access to both intances for akeneo user.
```

File: cloud_edition/flexibility_mode/docs/environments_access.rst [hunk 2]
```diff
@@ -102,3 +102,44 @@ If **akeneo**, as an SSH user or as a PIM process, creates files in the SFTP sub
...
 .. _`GitHub Help Center`:  https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent
 .. _`Akeneo Help Center`:  https://help.akeneo.com/portal/articles/access-akeneo-flexibility.html?utm_source=akeneo-docs&utm_campaign=flexibility_partner_starterkit
+
+
+Copy data from one instance to other
+************************************
+
+**Scenario:**
+
+    `User` want to transfer a SQL dump file from prod instance toward staging instance:
```
Comment (LudovicTOURMAN) [inline cloud_edition/flexibility_mode/docs/environments_access.rst:112]: Backquotes are not interpreted in this block

Comment (rybus) [inline cloud_edition/flexibility_mode/docs/environments_access.rst:112]:
```suggestion
    `User` wants to transfer a SQL dump file from production instance their staging instance:
```

File: cloud_edition/flexibility_mode/docs/environments_access.rst [hunk 3]
```diff
@@ -102,3 +102,44 @@ If **akeneo**, as an SSH user or as a PIM process, creates files in the SFTP sub
...
+    
+.. note::
+
+    **`User`** ssh connect with ssh-agent forwarding towards
```
Comment (rybus) [inline cloud_edition/flexibility_mode/docs/environments_access.rst:116]:
```suggestion
    **`User`** connects with SSH and forwards their local ssh-agent towards
```

File: cloud_edition/flexibility_mode/docs/environments_access.rst [hunk 4]
```diff
@@ -102,3 +102,44 @@ If **akeneo**, as an SSH user or as a PIM process, creates files in the SFTP sub
...
+        > **`project.akeneo.cloud.com`**, and run scp command towards
```
Comment (rybus) [inline cloud_edition/flexibility_mode/docs/environments_access.rst:117]:
```suggestion
        > **`project.akeneo.cloud.com`**, and runs scp command towards
```

File: cloud_edition/flexibility_mode/docs/environments_access.rst [hunk 5]
```diff
@@ -102,3 +102,44 @@ If **akeneo**, as an SSH user or as a PIM process, creates files in the SFTP sub
...
+
+    eval `ssh-agent`
+    ssh-add ~/.ssh/id_rsa
+    ssh -tA akeneo@project-staging.cloud.akeneo.com
+    
+    scp dump.sql akeneo@project:dump.sql
+
+.. warning::
+
+    For the scp step, note that no domain is mentioned.
```
Comment (rybus) [inline cloud_edition/flexibility_mode/docs/environments_access.rst:143]:
```suggestion
    For the scp step, please note that no domain is specified.
```

File: cloud_edition/flexibility_mode/docs/environments_access.rst [hunk 6]
```diff
@@ -102,3 +102,44 @@ If **akeneo**, as an SSH user or as a PIM process, creates files in the SFTP sub
 .. _`specific tasks`: ./partners.html
 .. _`GitHub Help Center`:  https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent
 .. _`Akeneo Help Center`:  https://help.akeneo.com/portal/articles/access-akeneo-flexibility.html?utm_source=akeneo-docs&utm_campaign=flexibility_partner_starterkit
+
+
+Copy data from one instance to other
```
Comment (rybus) [inline cloud_edition/flexibility_mode/docs/environments_access.rst:107]:
```suggestion
Copy data from one instance to another
```

**Review by adesaegher — APPROVED** [review]

File: cloud_edition/flexibility_mode/docs/environments_access.rst (hunk 2 above)
Comment (jagirard) [reply cloud_edition/flexibility_mode/docs/environments_access.rst]: removed

File: cloud_edition/flexibility_mode/docs/environments_access.rst (hunk 3 above)
Comment (jagirard) [reply cloud_edition/flexibility_mode/docs/environments_access.rst]: done

File: cloud_edition/flexibility_mode/docs/environments_access.rst (hunk 4 above)
Comment (jagirard) [reply cloud_edition/flexibility_mode/docs/environments_access.rst]: done

File: cloud_edition/flexibility_mode/docs/environments_access.rst [hunk 7]
```diff
@@ -102,3 +102,41 @@ If **akeneo**, as an SSH user or as a PIM process, creates files in the SFTP sub
...
+
+    eval `ssh-agent`
+    ssh-add ~/.ssh/id_rsa
+    ssh -tA akeneo@project-staging.cloud.akeneo.com
+    
+    scp dump.sql akeneo@project:dump.sql
+
+.. warning::
+
+    For the scp step, please note that no domain is specified.
```
Comment (rybus) [inline cloud_edition/flexibility_mode/docs/environments_access.rst:140]:
```suggestion
    On the SCP command, please note that no domain is specified.
```

File: cloud_edition/flexibility_mode/docs/environments_access.rst [hunk 8]
```diff
@@ -102,3 +102,41 @@ If **akeneo**, as an SSH user or as a PIM process, creates files in the SFTP sub
...
+    
+    Set the short hostname of instance. (connect to the target server and run command : `hostname`)
```
Comment (rybus) [inline cloud_edition/flexibility_mode/docs/environments_access.rst:142]:
```suggestion
    Use the short host name of instance. Connect to the target server and run `hostname` to get this value.
```

File: cloud_edition/flexibility_mode/docs/environments_access.rst [hunk 9]
```diff
@@ -102,3 +102,41 @@ If **akeneo**, as an SSH user or as a PIM process, creates files in the SFTP sub
...
+    **User** connects with SSH and forwards their local ssh-agent towards
+        > **project.akeneo.cloud.com**, and runs scp command towards
+        
+        > **project-staging.akeneo.cloud.com**
+
+    👨‍💼 💻   ──────> 🔑  ──────> 🖥  ────── 🔑 ──────> 📂 ──────> 🖥
+
+**Prerequisites:**
+
+- Get ssh key access to both intances for akeneo user.
```
Comment (rybus) [inline cloud_edition/flexibility_mode/docs/environments_access.rst:125]:
```suggestion
- Get SSH key access to both intances for akeneo user.
```

**Review by rybus — APPROVED** [review]

**Pull request merged**